lib/http.rb

```diff
@@ -15,7 +15,16 @@
 require "http/response/body"
 require "http/response/parser"
 
-# HTTP should be easy
+# HTTP should be easy.
+#
+# Basic Usage
+#
+#     HTTP.post("https://server.com", options)
```

Comment:
This example looks like `options` is mandatory. And does not provide any clues on what is it. So it's better to remove it, as it's not mandatory for basic usage.

Reply:
OK.  However, no where else do you say that options are passed to the `get`, `post`, or `request` method.  When I was reading the doc, I had no idea how to set options.

Reply:
Yes, we do:

---

![Screenshot_20200116-190830_Firefox Preview](https://user-images.githubusercontent.com/94782/72550867-f6c9f500-3893-11ea-9c03-561349578bae.jpg)

---

Also API docs landing page is readme, that contains examples of usage. If you want to emphasize that there's optional argument with options- that's fine. You can do that in additional `@example`:

``` ruby
# @example POST JSON data
#     HTTP.post("https://example.com", {
#       :json => { :message => "Hello, world!" }
#     })
#
# @see Options
```